src/EPR.Calculator.Frontend/Controllers/LocalAuthorityDisposalCostsController.cs

```diff
@@ -1,78 +1,124 @@
-                new LocalAuthorityDisposalCost { Id = 28, ParameterYear = "2024-25", EffectiveFrom = new DateTime(2024, 8, 28, 10, 12, 30, DateTimeKind.Utc), CreatedBy = "Test User", CreatedAt = new DateTime(2024, 8, 28, 10, 12, 30, DateTimeKind.Utc), LapcapDataMasterId = 1, LapcapDataTemplateMasterUniqueRef = "WLS-PC", Country = "Wales", Material = "Paper or card", TotalCost = 30.04m },
-                new LocalAuthorityDisposalCost { Id = 29, ParameterYear = "2024-25", EffectiveFrom = new DateTime(2024, 8, 28, 10, 12, 30, DateTimeKind.Utc), CreatedBy = "Test User", CreatedAt = new DateTime(2024, 8, 28, 10, 12, 30, DateTimeKind.Utc), LapcapDataMasterId = 1, LapcapDataTemplateMasterUniqueRef = "WLS-PL", Country = "Wales", Material = "Plastic", TotalCost = 30.05m },
-                new LocalAuthorityDisposalCost { Id = 30, ParameterYear = "2024-25", EffectiveFrom = new DateTime(2024, 8, 28, 10, 12, 30, DateTimeKind.Utc), CreatedBy = "Test User", CreatedAt = new DateTime(2024, 8, 28, 10, 12, 30, DateTimeKind.Utc), LapcapDataMasterId = 1, LapcapDataTemplateMasterUniqueRef = "WLS-ST", Country = "Wales", Material = "Steel", TotalCost = 30.06m },
-                new LocalAuthorityDisposalCost { Id = 31, ParameterYear = "2024-25", EffectiveFrom = new DateTime(2024, 8, 28, 10, 12, 30, DateTimeKind.Utc), CreatedBy = "Test User", CreatedAt = new DateTime(2024, 8, 28, 10, 12, 30, DateTimeKind.Utc), LapcapDataMasterId = 1, LapcapDataTemplateMasterUniqueRef = "WLS-WD", Country = "Wales", Material = "Wood", TotalCost = 30.07m },
-                new LocalAuthorityDisposalCost { Id = 32, ParameterYear = "2024-25", EffectiveFrom = new DateTime(2024, 8, 28, 10, 12, 30, DateTimeKind.Utc), CreatedBy = "Test User", CreatedAt = new DateTime(2024, 8, 28, 10, 12, 30, DateTimeKind.Utc), LapcapDataMasterId = 1, LapcapDataTemplateMasterUniqueRef = "WLS-OT", Country = "Wales", Material = "Other", TotalCost = 30.08m },
-            ]);
-
-            return localAuthorityDisposalCosts;
+            var lapcapRunApi = configuration.GetSection(ConfigSection.LapcapSettings)
+                                                  .GetSection(ConfigSection.LapcapSettingsApi)
+                                                  .Value;
+
+            if (string.IsNullOrEmpty(lapcapRunApi))
+            {
+                // Handle the null or empty case appropriately
+                throw new ArgumentNullException(lapcapRunApi, "The API URL cannot be null or empty.");
+            }
+
+            var client = clientFactory.CreateClient();
+            client.BaseAddress = new Uri(lapcapRunApi);
+            var year = configuration.GetSection("LapcapSettings").GetSection("ParameterYear").Value;
```

Comment:
Can we use the config section constants instead of string literals?

Reply:
done this